src/TerminalFighter/MainMenu/MainMenu.cpp

```diff
@@ -6,17 +6,62 @@
 MainMenu::MainMenu(std::string font_path, int width, int height)
     : title_(
         "Terminal Fighter",
-        {255, 255, 255},
-        width / 4,
-        height / 5,
-        width / 2,
-        height / 4,
+        WHITE,
+        0.25 * width,
+        0.2 * height,
+        0.5 * width,
+        0.25 * height,
         font_path,
-        68) {
+        68)
+    , start_(
+        "Start",
+        WHITE,
+        0.45 * width,
+        0.6 * height,
+        0.1 * width,
+        0.1 * height,
+        font_path,
+        68)
+    , quit_(
+        "Quit",
+        GRAY,
+        0.45 * width,
+        0.7 * height ,
+        0.1 * width,
+        0.1 * height,
+        font_path,
+        68)
+    , current_selection_(START){
 }
 
 MainMenu::~MainMenu() {}
 
 void MainMenu::render(SDL_Renderer& renderer) {
     title_.render(renderer);
+    start_.render(renderer);
+    quit_.render(renderer);
+}
+
+void MainMenu::move_up_selection() {
+    current_selection_ = (2 + (current_selection_ + 1) % 2) % 2;
```

Comment:
This math isn't the easiest to reason through. Can we do this more readable with conditionals?